modules/snooker/src/main/ts/ephox/snooker/api/TableFill.ts

```diff
@@ -33,6 +33,7 @@ const replace = <K extends keyof HTMLElementTagNameMap>(cell: SugarElement, tag:
   return replica;
 };
 
+// eslint-disable-next-line @tinymce/prefer-fun
```

Comment:
Can you add a comment above this saying why we don't prefer fun here?

Reply:
The comment is already inside the function.